dubbo-common/src/main/java/com/alibaba/dubbo/config/annotation/Reference.java

```diff
@@ -63,9 +72,9 @@
 
     String cluster() default "";
 
-    int connections() default 0;
+    int connections() default -1;
```

Comment:
I wonder if these default value changes will have side effects , have you checked how dubbo treats the default value of each item?

Reply:
**The default value of the annotation will be filtered as null**, so the default value is best set to a value that does not need to be used. 
`-1 `  may be used by some attributes? Change the default value of the int type to `Integer.MIN_VALUE`?  But `-1 ` is shorter.